sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json

```diff
@@ -13,7 +13,7 @@
             "get": {
                 "summary": "Retrieve a configuration for Iot Hub devices and modules by it identifier.",
                 "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "Configuration_Get",
+                "operationId": "IoTHubService_Configuration_Get",
```

Comment:
Does the IoTHubService_ prefix add anything here? Does it change how the code gets generated?

Reply:
yes it does not generate multiple clients like it is now

Reply:
It generates IoTHubServiceClient under Generated/Operations folder